packages/manager/modules/pci/src/projects/project/quota/increase-request/translations/Messages_fr_FR.json

```
@@ -3,6 +3,13 @@
   "pci_projects_project_quota_increase_submit_label": "Confirmer",
   "pci_projects_project_quota_increase_cancel_label": "Annuler",
   "pci_projects_project_quota_increase_content": "Êtes-vous sûr de vouloir augmenter les quotas de votre projet <strong>{{ projectId }}</strong> ?",
+  "pci_projects_project_quota_increase_buy_success_message": "Le bon de commande pour l'ajout de crédits à votre projet est disponible  <a class=\"oui-link_icon\" target=\"_blank\" href=\"{{ billingUrl }}\">ici<span class=\"oui-icon oui-icon-external_link\" aria-hidden=\"true\"></span></a>",
```

Extra space character found here:

```suggestion
  "pci_projects_project_quota_increase_buy_success_message": "Le bon de commande pour l'ajout de crédits à votre projet est disponible <a class=\"oui-link_icon\" target=\"_blank\" href=\"{{ billingUrl }}\">ici<span class=\"oui-icon oui-icon-external_link\" aria-hidden=\"true\"></span></a>",
```